tensorflow/python/ops/math_ops.py

```diff
@@ -100,6 +100,7 @@
 @@complex
 @@conj
 @@imag
+@@arg
```

Comment:
I'd argue we should perhaps only put this in tf.math.arg rather than in the root namespace. It doesn't seem that commonly used.